frontend/ui/widget/menu.lua

```diff
@@ -1449,6 +1423,22 @@ function Menu:onSwipe(arg, ges_ev)
     end
 end
 
+function Menu:setTitleBarIconText(extra_button_icon, title_text)
+    if self.extra_button and extra_button_icon then
+        self.extra_button:setIcon(extra_button_icon)
+    end
+    if self.menu_title and title_text then
+        self.menu_title:setText(title_text)
+    end
+    UIManager:setDirty(self.show_parent, "ui")
+end
+
+function Menu:onExtraButtonTap() -- override in the caller
```

Comment:
`-- to be overriden and implemented by the caller`

(I thought these could check and make use of a provided callback if any, but it seems all callers indeed do override these as methods.)

Reply:
`free()` tested:
Button: `setText` in Menu (page info in the bottom)
Button: `setIcon` in SortWidget
IconButton: `setIcon` in Menu (new left icon)